test/postControllerTest.js

```diff
@@ -0,0 +1,96 @@
+/* global it, describe */
+const express = require('express');
+const request = require('supertest');
+const postController = require('../controllers/postController');
+const sinon = require('sinon');
+const postService = require('../services/postService');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(express.static('../public'));
+
+app.use('', postController);
+const server = app.listen(3000);
+
+
+describe('postController', () => {
+  describe('GET /', () => {
+    it('should respond with homepage', (done) => {
+      const stub = sinon.stub(postService, 'getAllPosts').resolves('expected');
+      request(app)
+        .get('/')
+        .expect(200)
+        .expect('Content-Type', 'text/html; charset=utf-8')
+        .expect(/My First Blog/)
+        .end((err) => {
+          stub.restore();
+          if (err) return done(err);
+          return done();
+        });
+    });
+  });
+  describe('GET /posts', () => {
+    it('should respond with homepage', (done) => {
+      const stub = sinon.stub(postService, 'getAllPosts').resolves('expected');
+      request(app)
+        .get('/posts')
+        .expect(200)
+        .expect('Content-Type', 'text/html; charset=utf-8')
+        .expect(/My First Blog/)
+        .end((err) => {
+          stub.restore();
```

Comment:
This block seems to be pretty common... 
You should move this into a [Mocha hook](https://mochajs.org/#hooks)